pkg/controllers/work/apply_controller_test.go

```diff
@@ -444,7 +514,7 @@ func TestApplyUnstructured(t *testing.T) {
 			} else {
 				assert.Truef(t, err == nil, "err is not nil for Testcase %s", testName)
 				assert.Truef(t, applyResult != nil, "applyResult is not nil for Testcase %s", testName)
-				assert.Equalf(t, testCase.resultSpecHash, applyResult.GetAnnotations()[manifestHashAnnotation],
+				assert.Equalf(t, testCase.resultSpecHash, applyResult.GetAnnotations()[ManifestHashAnnotation],
```

Comment:
should we add the validation of the lastAppliedConfig?

Reply:
Checking lastAppliedConfig is difficult cause it contains live fields